node/cmd/guardiand/adminclient.go

```diff
@@ -375,6 +384,26 @@ func runSendObservationRequest(cmd *cobra.Command, args []string) {
 	}
 }
 
+func runDumpRPCs(cmd *cobra.Command, args []string) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	conn, c, err := getAdminClient(ctx, *clientSocketPath)
+	if err != nil {
+		log.Fatalf("failed to get admin client: %v", err)
+	}
+	defer conn.Close()
+
+	resp, err := c.DumpRPCs(ctx, &nodev1.DumpRPCsRequest{})
+	if err != nil {
+		log.Fatalf("failed to run dump-rpcs: %s", err)
+	}
+
+	for node, rpc := range resp.Response {
```

Comment:
Extreme nit: The key is not really a node, it's a parameter (and in some cases there are more than one parameter per node).

Reply:
Fixed